cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLUtil.java

```diff
@@ -33,27 +35,45 @@ public class CloudSQLMySQLUtil {
    * @param connectionName Connection Name for the CloudSQL MySQL instance
    */
   public static void checkConnectionName(
-      FailureCollector failureCollector, String instanceType, String connectionName) {
+    FailureCollector failureCollector, @Nullable String instanceType, @Nullable String connectionName) {
+
+    if (Strings.isNullOrEmpty(instanceType)) {
```

Comment:
So if this cannot be null or empty, can we mark these as required fields?

Reply:
ack